### PR TITLE
Add howto for signing Focus for Android

### PR DESCRIPTION
**Pull request by JohanLorenzo**

### CODE REVIEW

File: how-tos/focus_for_android.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+1. Verify signatures: `jarsigner -verify -verbose -keystore /builds/signing/rel-key-signing-server/secrets/focus-jar app-focus-webkit-release-unsigned.apk focus`. Repeat for Klar.
+1. `mv app-focus-webkit-release-unsigned.apk app-focus-webkit-release-signed.apk`
+1. Fetch signed APKs on your local machine.
+
+## Optimize APKs
+
+Google Play refuses non-optimized APKs. The signature changes the structure of the APK archive, which breaks the Zip optimization.
+
+1. Install the latest Android SDK, to get the build tools. For instance, on Mac: `brew cask install android-sdk` (requires `brew tap caskroom/cask`)
+1. `/opt/android-sdk/build-tools/26.0.1/zipalign -v 4 app-klar-webkit-release-signed.apk app-klar-webkit-release-signed-aligned.apk`. Repeat for Klar.
```
Comment (MihaiTabara) [inline how-tos/focus_for_android.md]: Nit: If installed via homebrew, I think it automatically links it under `/usr/local/bin/`. At least it did on my Mac.

**Review by MihaiTabara — APPROVED** [review]
Am not hijacking this from @kmoir but since I'm here, from my standpoint it's a r+ for this! thanks for putting them up! :shipit:

**Comment by JohanLorenzo** [comment]
We'll be able to test the docs. A new signing request just came in [bug 1394372](https://bugzilla.mozilla.org/show_bug.cgi?id=1394372) (I thought it was gonna be next month, but I was wrong). cc @kmoir

File: how-tos/focus_for_android.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Manually sign Focus for Android APKs
+
+There are 2 APKs: one is Focus, the other for the german-speaking population: Klar.
+
+## Create signatures
+1. `ssh signing4.srv.releng.scl3.mozilla.com`
+1. Download unsigned APKs
+1. `jarsigner -keystore /builds/signing/rel-key-signing-server/secrets/focus-jar app-focus-webkit-release-unsigned.apk focus`. Repeat for Klar: only change the name of the APK, the certificate alias (`focus`) remains identical.
```
Comment (kmoir) [inline how-tos/focus_for_android.md]: You'll be prompted for the focus passphrase which is stored in the releng private repo.

Comment (kmoir) [reply how-tos/focus_for_android.md]: You'll also get a warning "Warning: 
No -tsa or -tsacert is provided and this jar is not timestamped. Without a timestamp, users may not be able to validate this jar after the signer certificate's expiration date (2044-10-25) or after any future revocation date."

File: how-tos/focus_for_android.md (hunk 1 above)
Comment (kmoir) [reply how-tos/focus_for_android.md]: The line above says "repeat for klar" but actually already refers to the klar binaries.  Also a +1 to Mihai's comments above.

File: how-tos/focus_for_android.md (hunk 2 above)
Comment (MihaiTabara) [reply how-tos/focus_for_android.md]: nice catch @kmoir !

**Comment by JohanLorenzo** [comment]
Thanks for the reviews! I addressed your comments

**Pull request merged**